### PR TITLE
make stores Closable

### PR DESCRIPTION
**Pull request by sprsquish**
Extend stores (and related parts) with Closable. This is the first and smallest step toward moving Zipkin to TwitterServer.

### CODE REVIEW

**Comment by johanoskarsson** [comment]
+1

File: zipkin-cassandra/src/main/scala/com/twitter/zipkin/storage/cassandra/CassandraAggregates.scala [hunk 1]
```diff
@@ -42,7 +42,7 @@ case class CassandraAggregates(
   dependenciesCF: ColumnFamily[ByteBuffer, Long, gen.Dependencies]
 ) extends Aggregates {
 
-  def close() {
+  def close(deadline: Time): Future[Unit] = FuturePool.unboundedPool {
```
Comment (bmdhacks) [inline zipkin-cassandra/src/main/scala/com/twitter/zipkin/storage/cassandra/CassandraAggregates.scala:45]: Doesn't the anorm code already have it's own future pool?

**Comment by bmdhacks** [comment]
I'm surprised none of these other storages like redis and cassandra have Closable compatible close methods.

File: zipkin-cassandra/src/main/scala/com/twitter/zipkin/storage/cassandra/CassandraAggregates.scala (hunk 1 above)
Comment (sprsquish) [inline zipkin-cassandra/src/main/scala/com/twitter/zipkin/storage/cassandra/CassandraAggregates.scala:45]: ah.. didn't see that.

Comment (bmdhacks) [inline zipkin-cassandra/src/main/scala/com/twitter/zipkin/storage/cassandra/CassandraAggregates.scala:45]: Hbase might have one too... I haven't looked.

Comment (sprsquish) [inline zipkin-cassandra/src/main/scala/com/twitter/zipkin/storage/cassandra/CassandraAggregates.scala:45]: They're private to the Mapper trait.

**Comment by sprsquish** [comment]
Redis and Cassie are behind the times, by quite a bit. They don't implement Closable, yet. And updating them isn't a yak I want to shave just yet.

**Comment by sprsquish** [comment]
took this a different direction

**Pull request closed without merging**